irohad/ametsuchi/impl/flat_file/flat_file.cpp

```diff
@@ -102,22 +104,26 @@ nonstd::optional<Identifier> check_consistency(const std::string &dump_dir) {
 
   auto n = static_cast<uint32_t>(status);
   tmp_id++;
+
+  // get last available identificator
```

Comment:
Better to write "last available identifier" because there is describes invariant for the variable, not an action.